### ISSUE TITLE
Makefile fails if abspath contain spaces

### ISSUE THREAD

**Issue by meling** [bufbuild/connect-go#506]
**Describe the bug**

If the path to your connect-go repo has one or more spaces, it fails to build due to a [well-known bug in Make](https://savannah.gnu.org/bugs/?712).

Example path: `/Users/meling/Team-UIS Dropbox/Hein Meling/work/connect-go/`

**Fix / Workaround**

Surround relevant paths with quotes. It is such a small change that perhaps some of the devs could fix this rather than me raising a PR... But I can also send a PR, if you prefer. See diff below:

```diff
diff --git a/Makefile b/Makefile
index a872b47..2bfd6ab 100644
--- a/Makefile
+++ b/Makefile
@@ -54,7 +54,7 @@ lintfix: $(BIN)/golangci-lint $(BIN)/buf ## Automatically fix some lint errors
 .PHONY: generate
 generate: $(BIN)/buf $(BIN)/protoc-gen-go $(BIN)/protoc-gen-connect-go $(BIN)/license-header ## Regenerate code and licenses
 	rm -rf internal/gen
-	PATH=$(abspath $(BIN)) $(BIN)/buf generate
+	PATH="$(abspath $(BIN))" $(BIN)/buf generate
 	$(BIN)/license-header \
 		--license-type apache \
 		--copyright-holder "Buf Technologies, Inc." \
@@ -76,19 +76,19 @@ $(BIN)/protoc-gen-connect-go:

 $(BIN)/buf: Makefile
 	@mkdir -p $(@D)
-	GOBIN=$(abspath $(@D)) $(GO) install github.com/bufbuild/buf/cmd/buf@v1.13.1
+	GOBIN="$(abspath $(@D))" $(GO) install github.com/bufbuild/buf/cmd/buf@v1.13.1

 $(BIN)/license-header: Makefile
 	@mkdir -p $(@D)
-	GOBIN=$(abspath $(@D)) $(GO) install \
+	GOBIN="$(abspath $(@D))" $(GO) install \
 		  github.com/bufbuild/buf/private/pkg/licenseheader/cmd/license-header@90fa81df0e9ef86ed505956be1ab1e8ccd49aa52

 $(BIN)/golangci-lint: Makefile
 	@mkdir -p $(@D)
-	GOBIN=$(abspath $(@D)) $(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.0
+	GOBIN="$(abspath $(@D))" $(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.0

 $(BIN)/protoc-gen-go: Makefile go.mod
 	@mkdir -p $(@D)
 	@# The version of protoc-gen-go is determined by the version in go.mod
-	GOBIN=$(abspath $(@D)) $(GO) install google.golang.org/protobuf/cmd/protoc-gen-go
+	GOBIN="$(abspath $(@D))" $(GO) install google.golang.org/protobuf/cmd/protoc-gen-go
```

**Additional context**

Sadly, Dropbox also has a long standing issue regarding this that they refuse to fix; specifically they insist on using spaces in team folders, and more recently, they also added the user's full name with spaces to the path.

**Comment by bufdev** [comment]
Sorry about that! Fixing in https://github.com/bufbuild/connect-go/pull/508.

**Comment by bufdev** [comment]
This is fixed in #508 - thanks for your patience!
